pytorch_lightning/tuner/tuning.py

```diff
@@ -43,6 +44,10 @@ def _tune(
 
         self.trainer.strategy.connect(model)
 
+        is_tuning = self.trainer.auto_scale_batch_size or self.trainer.auto_lr_find
+        if self.trainer._accelerator_connector.is_distributed and is_tuning:
+            raise MisconfigurationException("`trainer.tune()` is not supported with a distributed strategy.")
```

Comment:
Could probably add the strategy being used.

Reply:
good point!!